config/environments.go

```diff
@@ -86,6 +86,12 @@ func (c *Config) Environment() DatabricksEnvironment {
 			if v.azureEnvironment.Name != azureEnv {
 				continue
 			}
+			if strings.HasPrefix(v.dnsZone, ".dev") || strings.HasPrefix(v.dnsZone, ".staging") {
```

Comment:
Correct, this actually failed in our nightlies so we can just remove this as I added it in before the v0.26 release.

Reply:
i think we should keep the impl for internal users